### PR TITLE
[IA-2923] Add CI scripts for terra-jupyter-gatk-ovtf image

### PR DESCRIPTION
**Pull request by rtitle**
JIRA: https://broadworkbench.atlassian.net/browse/IA-2923
Related PR: https://github.com/DataBiosphere/terra-docker/pull/243

This adds CI and testing (stubbed out for now) for the `terra-jupyter-gatk-ovtf` image. The idea is to add it to the "Community Images" section in Terra UI once fully tested.

Once this PR is merged the image should be built and pushed at: `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk-ovtf:0.1.0`. It should also get automatically rebuilt with a patch version bump once a parent image (such as `terra-jupyter-python` is updated).

cc @ravi9, will note the places where tests need to be added.

### CODE REVIEW

File: terra-jupyter-gatk-ovtf/tests/smoke_test.ipynb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "## Test OVTF"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"TODO add test cases\")"
```
Comment (rtitle) [inline terra-jupyter-gatk-ovtf/tests/smoke_test.ipynb:52]: Can add one or more notebook cells as test cases.

File: terra-jupyter-gatk-ovtf/tests/smoke_test.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+TODO(all): Add additional tests and/or tests with particular assertions, as we encounter Python package version
+incompatibilities not currently detected by these tests.
+"""
+
+import os
+import pytest
+
+def test_ovtf():
+  # TODO
```
Comment (rtitle) [inline terra-jupyter-gatk-ovtf/tests/smoke_test.py:14]: Can add `pytest`cases here. The GitHub action will execute these and report on failures.

File: terra-jupyter-gatk-ovtf/tests/ovtf_smoke_test.ipynb [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    "# Package versions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 freeze"
```
Comment (Qi77Qi) [inline terra-jupyter-gatk-ovtf/tests/ovtf_smoke_test.ipynb:29]: any particular reason we're testing `freeze` here?

**Review by Qi77Qi — APPROVED** [review]

File: terra-jupyter-gatk-ovtf/tests/ovtf_smoke_test.ipynb (hunk 3 above)
Comment (rtitle) [reply terra-jupyter-gatk-ovtf/tests/ovtf_smoke_test.ipynb:29]: I think it's to just log all the pip packages for debugging info

**Comment by rtitle** [comment]
Thanks for looking @Qi77Qi. I'll leave this open so @ravi9 can review as well.

@ravi9 let me know whether you'd prefer to add the test cases directly to this PR; or if this can be merged and you can open follow-up PRs for subsequent changes.

**Comment by rtitle** [comment]
Looks like it passed :) Output notebook: https://github.com/DataBiosphere/terra-docker/suites/3632645846/artifacts/87947025

**Comment by rtitle** [comment]
Going to merge this, will watch the Jenkins build for pushing the image to `us.gcr.io/broad-dsp-gcr-public/...`

**Comment by rtitle** [comment]
OK! The image is now pushed to:  us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk-ovtf:0.1.0

Feel free to give it a try, and feel free to make subsequent PRs to this repo for any changes/fixes (and update the CHANGELOG). 

Thanks!

**Comment by ravi9** [comment]
Thanks very much @rtitle !
We tested on Terra and everything works as expected. Updated the readme and sent a PR #248 

Another quick request: Could you also include this image in Terra drop-down under community images ?

**Comment by rtitle** [comment]
Sure, will look at adding it to Terra UI today

**Pull request merged**